en/configure-tidb-binlog-drainer.md

```diff
@@ -6,7 +6,7 @@ aliases: ['/docs/tidb-in-kubernetes/dev/configure-tidb-binlog-drainer/']
 
 # TiDB Binlog Drainer Configurations on Kubernetes
 
-This document introduces the configuration parameters for a TiDB Binlog drainer on Kubernetes.
+This document introduces the configuration parameters for a [TiDB Binlog](deploy-tidb-binlog.md) drainer on Kubernetes. Note that TiDB Binlog is deprecated starting from TiDB v8.3.0.
```

Comment:
I would make it clearer that deprecation started in v7.5.0 and is fully deprecated in v8.3.0 and will be removed in a future release.

I would propose a different last sentence, something like:
> Please note that TiDB Binlog replication was deprecated in v7.5.0, and TiDB Binlog will be fully deprecated starting from v8.3.0, with removal planned for a future release.